Tests/test_util.py

```diff
@@ -39,6 +54,22 @@ def test_is_not_path(tmp_path):
     assert not it_is_not
 
 
+def test_path_like_file_is_not_path(tmp_path):
+    # Arrange
+    import io
+
+    class PathAndFileThingy(io.TextIOWrapper):
+        def __fspath__(self):
+            return self.name
+
+    with PathAndFileThingy(open(tmp_path / "temp.ext", "wb")) as test_obj:
+        # Act
+        it_is_not = _util.isPath(test_obj)
+
+        # Assert
+        assert not it_is_not
```

Comment:
```suggestion
    with open(tmp_path / "temp.ext", "wb") as fp:
		with PathAndFileThingy(fp) as test_obj:
			# Act
			it_is_not = _util.isPath(test_obj)

			# Assert
			assert not it_is_not
```
Adds a context manager for the file pointer.

Reply:
`io.TextIOBase` takes ownership of the provided binary I/O object, so it will be automatically closed when closing the “PathAndFileThingy” at the end of the with-block.